target/consensource-ui
# Begin txn tests

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

Tests for the `transaction` service

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `npm run test src/services/transaction/transaction.test.ts` and confirm all tests pass
- `npm run test` and confirm all tests pass
